tests/features/payments.feature

```diff
@@ -48,79 +39,68 @@ Scenario: Update a payment (PUT)
 
 Scenario: Update a payment with bad or no data (PATCH)
     When I get "payments" with id "1"
-    Then I should see a payment with id "1" and "nickname" = "my credit"
+    Then I should see a payment with id "1" and "nickname" = "cashmoney"
     When I change "nicknam3" to "new credit", but misspell the attribute
-    And I patch "payments" with id "1"
+    And I patch "payments" with id "1" with bad data
```

Comment:
Maybe we can change this wording to be `When I change my payment "1" with bad data`.  It sounds a little more user-like.